### ISSUE TITLE
FinQA pretrained model

### ISSUE THREAD

**Issue by ghost** [NeuraSearch/NeurIPS-2022-Submission-3358#1]
Hello.. Seems like there is a missing checkpoint_best_0.65.pt file from the published pretrained FinQA model.  Also, is there a process_data directory missing?

**Comment by KnightZhang625** [comment]
Hi, there. 
1. You can use the processed data under datasets/cached_data/, which means you could use the model or retrain the model directly.
2. The download link for the published pre-trained FinQA model could be found in README, which is a google drive link.
Please contact if you have any furthre problem. Thanks!

**Comment by ghost** [comment]
Hi @KnightZhang625,

To clarify my issues above:

I think a directory of python code called process_data is missing.  The reference to it is in model_wrapper.py on line 16.  

I downloaded the FinQA model zip, but the file checkpoint_best_0.65.pt does not exist in the zip.

Thanks, Ian

**Comment by KnightZhang625** [comment]
Hi, @ianupright .
Sorry for that.

1.  I just found the problem, and commented out the line in model_wrapper.py at line 16, which is actually a useless import, having no influence with executing the code.
2. For the FinQA model zip, I do find that "checkpoint_best_0.65.pt" is missing. I will go to find it on server and will upload it once I found it. 

Apologize for any inconvenience!

Cheers,
Jiaxin

**Comment by ghost** [comment]
Thanks @KnightZhang625,

Looking further, it would be great to see and understand the data processing code.  The start_process.sh script calls it but I think the code is not there?

Thanks, Ian

**Comment by KnightZhang625** [comment]
Hi, @ianupright .
 
The missing .pt could be found in the link now. However, since the policy of school server stipulates that the limitation of the storage. I couldn't find the original .pt file trained in May, before the NeurIPS 2022 deadline. As a result, I quickly re-trained the model, the metric scores on FinQA test dataset is slightly different with the reported one.

Cheers,
Jiaxin

**Comment by ghost** [comment]
Hi @KnightZhang625.

One reason to see the data processing code:

For MathQA I see this in the cached_dev_data.json:
 "question_passage": "15.06  0.00001  ",

But in the raw_data/mathqa/dev.json:
"Problem": "15.06 * 0.00001 = ?",

Thanks, Ian

**Comment by KnightZhang625** [comment]
Hi, @ianupright. I will release the data processing code once I finish organizing them. This is because the data processing code contains lots of experiments relevant to other datasets, and it is totally messy. Thanks!
